setup.cfg

```diff
@@ -14,3 +14,4 @@ addopts =
   --color=yes
 testpaths = payments
 DJANGO_SETTINGS_MODULE = test_settings
+pythonpath = .
```

Comment:
Why is this here?

Reply:
This was need to resolve the problem in https://github.com/jazzband/django-payments/actions/runs/3783046529, where all test suites fail because django cannot find the settings file. I don't know it started, but this solves the problem in a clean way. If you have a better way to resolve it, please do.

Reply:
@WhyNotHugo is this okay?

Reply:
Yeah, it was previously not required and I can't figure out why now it is. I don't see anything in tox's changelog that would result in this being needed now.